xml/nodes-3.10.rng

```diff
@@ -10,7 +10,12 @@
       <zeroOrMore>
         <element name="node">
           <attribute name="id"><text/></attribute>
-          <attribute name="uname"><text/></attribute>
+          <optional>
+            <attribute name="cluster-id"><text/></attribute>
```

Comment:
Do we need a feature-set bump for this? There's a possible incompatibility if one node has `cluster-id` and the other doesn't -- but if neither does, it's fine, since we'll default to `id`.

Reply:
The schema can't be upgraded until the DC supports it, so cluster-id can't be used and uname can't be omitted until then. If the DC would treat a CIB that's valid under both schemas differently, then we'd need a bump.

Reply:
Ah, yep.